docs/workflow/testing/libraries/testing.md

```diff
@@ -1,54 +1,76 @@
 # Testing Libraries
 
-We use the OSS testing framework [xunit](https://github.com/xunit/xunit).
+## Full Build and Test Run
 
-To build the tests and run them you can call the libraries build script. For libraries tests to work, you must have built the coreclr or mono runtime for them to run on.
+These example commands automate the test run and all pre-requisite build steps in a single command from a clean enlistment.
 
-**Examples**
-- The following shows how to build only the tests but not run them:
+- Build all pre-requisites (clr + libs) and tests and run them using CoreCLR:
 ```
```

Comment:
This will build debug CoreCLR flavor that is super slow. `-rc Release` should be the first and recommended option here. It is what most libraries devs use.

Reply:
Done, examples with Release at the top and warning added on the debug example indicating it is very slow.